### PR TITLE
Implement pages

### LINKED ISSUE #1: [3pt] food list (with all CRUD implementation, except for 'update')
### Food list:

- Should display a list of food added by the logged-in user as in the wireframe (for teams with only 2 people - display also quantity of a given food).
- Should lead to a form that allows users to add new food.

### LINKED ISSUE #7: [3pt] Navigation menu
- Create a navigation menu that allows users to open all of the pages you created.

### PR DESCRIPTION
**Pull request by maen-al-khraisha**
Add nav bar. 
Add food list.
Add food form.

### CODE REVIEW

File: app/views/foods/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+<div class="conatiner-fluid shadow m-5 p-5">
```
Comment (bobb-Rob) [inline app/views/foods/index.html.erb]: Well done on this. 

I just want to point out this class name on line 2 of this `app/views/foods/index.html.erb` file. Shouldn't it be container?

If not, then it is okay.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @maenkhraisha , 

Good job so far! 👏👏👏 
There are some linter issues that still need to be worked on,  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights

✅ Great job
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

File: app/views/foods/index.html.erb (hunk 1 above)
Comment (maen-al-khraisha) [reply app/views/foods/index.html.erb]: Its personal preference , but its ok I will change it

**Review by bobb-Rob — APPROVED** [review]
Great work!

**Pull request merged**